src/Auth/Startup.cs

```diff
@@ -44,27 +50,42 @@ public void ConfigureServices(IServiceCollection services)
                 .AddEntityFrameworkStores<ApplicationDbContext>()
                 .AddDefaultTokenProviders();
 
-            // Register the OpenIddict services, including the default Entity Framework stores.
-            services.AddOpenIddict<ApplicationUser, ApplicationDbContext>().Configure(options =>
+            // add OpenIddict
+            services.AddOpenIddict<ApplicationUser, ApplicationDbContext>()
+                .DisableHttpsRequirement()
+                .UseJsonWebTokens();
+
+            //Add Set the Admin Credentials from appsettings to a POCO Object
+            services.Configure<LoginViewModel>(myoptions =>
             {
-                // During development, you can disable the HTTPS requirement.
-                options.AllowInsecureHttp = true;
-                options.UseJwtTokens();
-            });                
+                myoptions.Email = Configuration.GetSection("AppCredentials:Admin:Username").Value;
+                myoptions.Password = Configuration.GetSection("AppCredentials:Admin:Password").Value;
+            });
 
             services.AddMvc();
 
             // Add application services.
             services.AddTransient<IEmailSender, AuthMessageSender>();
             services.AddTransient<ISmsSender, AuthMessageSender>();
+
+            //swagger configurations
+            services.AddSwaggerGen();
```

Comment:
We can skip swagger for hip-auth cause it doesn't picks up the main login end point and we have to document it, so we can add register end point there as well, no need for whole swagger just for two urls